scripts/postbuild.js

```diff
@@ -34,15 +42,16 @@ const main = async () => {
   rename('dist/style.css', 'dist/prime.css');
 
   // Add Typescript definitions to the library directory
+  // TODO(mc, 2023-03-23): this file is empty, this won't work
   copy('src/prime.d.ts', 'dist/prime.d.ts');
```

Comment:
I'd like prime to export types, it just hasn't been set up, but to do this we'd likely need to patch into vue's type checking for HTML elements. Since that might be a lot of work, and we're considering migrating to svelte and all of that work would therefore be undone, I'd rather punt on it